firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     implementation project(':firebase-components')
     implementation project(':firebase-installations-interop')
     implementation project(':firebase-common')
+    testImplementation project(path: ':firebase-appdistribution')
```

Comment:
I needed to add this for my IDE to recognize the classes from my tests. Don't know why.